easybuild/easyconfigs/f/fast5/fast5-0.6.5.eb

```diff
@@ -0,0 +1,33 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+# Fred Hutchinson Cancer Research Institute, Seattle US
+
+easyblock = 'Tarball'
+
+name = 'fast5'
+version = '0.6.5'
+
+homepage = 'http://simpsonlab.github.io/2017/02/27/packing_fast5/'
+description = """A lightweight C++ library for accessing Oxford Nanopore
+ Technologies sequencing data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/mateidavid/fast5/archive']
+sources = ['v%(version)s.zip']
+checksums = ['c7a052cc125b21d6ade1cd63af992b8e63a9857019359ad9e9b55234f347992f']
+
+postinstallcmds = ['mv %(installdir)s/src %(installdir)s/include']
+
+sanity_check_paths = {
+    'files': ['include/fast5.hpp'],
+    'dirs': ['include'],
+}
+
+modextrapaths = {
+    'LD_LIBRARY_PATH': 'include',
```

Comment:
isn't this a header only library?

Reply:
Yes, header only.  How does g++ find this header files if they are not in LD_LIBRARY_PATH?

Reply:
`LD_LIBRARY_PATH` is where the linker looks for libraries, for the compiler to find headers you can add the `include` folder to `CPATH`, or  `CPLUS_INCLUDE_PATH`